src/main/scala/dpla/ingestion3/model/ModelConverter.scala

```diff
@@ -141,4 +144,7 @@ object ModelConverter {
   private[model] def uriSeq(row: Row, fieldPosition: Integer): Seq[URI] =
     stringSeq(row, fieldPosition).map(new URI(_))
 
+  private[model] def toOptionJvalue(row: Row, fieldPosition: Integer): Option[JValue] = {
```

Comment:
This shouldn't block merging, but JValues can be JNothing, which might be easier to deal with than an Option, since you can run all the same path walking `\` things on JNothing. They just don't return results.